DynamicPackagingVODContent/StreamHLSAndDASH/src/main/java/sample/StreamHLSAndDASH.java

```diff
@@ -123,57 +131,68 @@ private static void runStreamHlsAndDash(ConfigWrapper config) {
                 System.out.println();
 
                 // Now that the content has been encoded, publish it for Streaming by creating
-                // a StreamingLocator. 
-                StreamingLocator locator = getStreamingLocator(manager, config.getResourceGroup(), config.getAccountName(),
+                // a StreamingLocator.
+                StreamingLocator locator = getStreamingLocator(manager,
+                        config.getResourceGroup(), config.getAccountName(),
                         outputAsset.name(), locatorName);
 
                 StreamingEndpoint streamingEndpoint = manager.streamingEndpoints()
-                        .getAsync(config.getResourceGroup(), config.getAccountName(), STREAMING_ENDPOINT_NAME)
-                        .toBlocking().first();
+                        .get(config.getResourceGroup(), config.getAccountName(),
+                                STREAMING_ENDPOINT_NAME);
 
                 if (streamingEndpoint != null) {
                     // Start The Streaming Endpoint if it is not running.
                     if (streamingEndpoint.resourceState() != StreamingEndpointResourceState.RUNNING) {
-                        manager.streamingEndpoints().startAsync(config.getResourceGroup(), config.getAccountName(), STREAMING_ENDPOINT_NAME).await();
+                        manager.streamingEndpoints().start(config.getResourceGroup(),
+                                config.getAccountName(), STREAMING_ENDPOINT_NAME);
 
                         // We started the endpoint, we should stop it in cleanup.
                         stopEndpoint = true;
                     }
 
-                    List<String> urls = getHlsAndDashStreamingUrls(manager, config.getResourceGroup(), config.getAccountName(), locator.name(), streamingEndpoint);
+                    List<String> urls = getHlsAndDashStreamingUrls(manager,
```

Comment:
Lets review the code in the getHLS and DASH function.  I would like to make sure this is matching more to what we do in the .NET and Typescript samples now.  

I chose in those samples to NOT use the list URLs methods, but instead build the CMAF URLs directly.